presto-main/src/main/java/com/facebook/presto/sql/planner/PlanFragmenter.java

```diff
@@ -520,6 +528,13 @@ public PlanNode visitExchange(ExchangeNode exchange, RewriteContext<FragmentProp
             }
         }
 
+        @Override
+        public PlanNode visitNativeExecution(NativeExecutionNode nativeExecution, RewriteContext<FragmentProperties> context)
```

Comment:
Is this method needed? Will plan fragmenter actually visit such node? Or it will just create them.